frontends/concrete-python/tests/execution/test_matmul.py

```diff
@@ -140,19 +149,43 @@ def lhs_function(x):
     @fhe.compiler({"x": "encrypted"})
     def rhs_function(x):
         return np.matmul(lhs_cst, x)
-
-    lhs_inputset = [np.random.randint(minimum, maximum, size=lhs_shape) for i in range(100)]
-    rhs_inputset = [np.random.randint(minimum, maximum, size=rhs_shape) for i in range(100)]
+    
+    lhs_inputset = [rgen(lhs_shape) for i in range(100)]
+    rhs_inputset = [rgen(rhs_shape) for i in range(100)]
```

Comment:
Here for example, I'd prefer seeing maximum and minimum. It's okay to be verbose in this case compared to jumping to rgen just to see what it's doing.